packages/suite/src/views/wallet/send/Outputs/Address.tsx

```diff
@@ -230,38 +257,52 @@ export const Address = ({ output, outputId, outputsCount }: AddressProps) => {
                     return translationString('RECIPIENT_IS_NOT_VALID');
                 }
             },
-            // Eth addresses are valid without checksum but Trezor displays them as checksummed.
-            checksum: async (address: string) => {
-                if (networkType === 'ethereum' && !checkAddressCheckSum(address)) {
+            evmchecks: async (address: string) => {
+                if (networkType === 'ethereum') {
                     if (isOnline) {
                         const params = {
                             descriptor: address,
                             coin: symbol,
                         };
-                        // 1. If the address is used but unchecksummed, then Suite will automatically
-                        // convert the address to the correct checksummed form and inform the user as described in the OP.
                         const result = await TrezorConnect.getAccountInfo(params);
```

Comment:
What if `result.success` is false? Before it would fall through to `TR_ETH_ADDRESS_CANT_VERIFY_HISTORY`, now nothing happens